spec/graphql/functions/create_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+describe Functions::Create do
+
+  describe '#call' do
+    context 'when all parameters are provided' do
+      context 'when model does not have a nested attributes' do
+        it 'creates a record' do
+          params = {
+            'customer' =>
+              {
+                'name' => 'Vivien Wilderman',
+                'webpage' => 'http://kozey.com/kelsie',
+                'notes' => 'Slow-carb poutine occupy cleanse leggings cray dreamcatcher taxidermy.'
+              }
+          }
+
+          result = Functions::Create.new(Order).call(nil, params, nil)
+
+          expect(result.persisted?).to be_truthy
+          expect(result).to have_attributes(
+            name: 'Vivien Wilderman',
+            webpage: 'http://kozey.com/kelsie',
+            notes: 'Slow-carb poutine occupy cleanse leggings cray dreamcatcher taxidermy.'
+          )
+        end
+      end
+
+
+      context 'when model have a nested attributes' do
+        it 'creates a record with a nested supplier' do
+          params = {
+            'product' =>
+              {
+                'product_code' => 6393283882,
+                'product_name' => 'Ergonomic Wooden Lamp',
+                'standard_cost' => 10.15,
+                'list_price' => 11.11,
+                'reorder_level' => 6056,
+                'target_level' => 5822,
+                'discontinued' => false,
+                'minimum_reorder_quantity' => 7040,
+                'category' => 'Electronics',
+                'supplier' =>
+                  {
+                    'name' => 'Supplier #1',
+                    'webpage' => 'http://example.com',
+                    'notes' => 'Something note'
```

Comment:
In line 57+ we have `product_code: '6393283882',` etc. Cant we define fields/values once and use it here with stringify keys and without it in expect. After all it should be the same.

This applies to all those tests.